src/types.ts

```diff
@@ -33,11 +25,18 @@ export type OpenRouterSharedSettings = {
       }
   );
 
-  extraBody?: Record<string, any>;
-
   /**
-A unique identifier representing your end-user, which can help OpenRouter to
-monitor and detect abuse.
+   * A unique identifier representing your end-user, which can
+   * help OpenRouter to monitor and detect abuse.
    */
   user?: string;
 };
+
+export type OpenRouterSharedSettings = OpenRouterProviderOptions & {
+  /**
+   * @deprecated use `reasoning` instead
+   */
+  includeReasoning?: boolean;
+
+  extraBody?: Record<string, any>;
```

Comment:
should we mark this as deprecated too? in favor of provider options?

Reply:
I don't think so -- it provide options for when user need it